gitbootcamp.rst

```diff
@@ -240,6 +240,27 @@ Commit the files:
 
    git commit -m "<message>"
 
+.. _diff-changes:
+
+Comparing Changes
+-----------------
+
+View all non-commited changes::
+
+   git diff
+
+Compare to the ``main`` branch::
+
+   git diff main
+
+Exclude generated files from diff (note the single quotes)::
+
+   git diff main ':(attr:!generated)'
+
+Exclude generated files from diff by default::
+
+   git config diff.generated.binary true
+
```

Comment:
Should we mention that both of these options are driven by the `generated` attribute in `.gitattributes`?

Reply:
Sounds good. PTAL.